assets/src/edit-story/output/page.js

```diff
@@ -95,7 +99,11 @@ function OutputPage({ page, autoAdvance, defaultPageDuration }) {
           </amp-story-grid-layer>
         )}
 
-        <amp-story-grid-layer template="vertical" aspect-ratio={ASPECT_RATIO}>
+        <amp-story-grid-layer
+          template="vertical"
+          aspect-ratio={ASPECT_RATIO}
+          class="grid-layer-fullbleed-area"
```

Comment:
Why use two different classes instead of one? Let's just name the class `grid-layer`.

Reply:
Got it. It is actually cleaner in this way. We don't need two classes.